spec/parse_messagepack_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'helper'
+require 'faraday_middleware/response/parse_message_pack'
```

Comment:
this require shouldn't be needed, but it is due to the issue in the autoloads above